docs/healthchecks.md

```diff
@@ -37,15 +43,54 @@ configured health check inside the container and reports the result. It serves a
 health checks, especially in scenarios where external scheduling is used.
 
 Example:
-
 ```
 nerdctl container healthcheck <container-id>
 ```
 
-### Future Work (WIP)
+## Automatic Health Checks with systemd
+
+On Linux systems with systemd, nerdctl automatically creates and manages systemd timer units to execute health checks at the configured intervals. This provides reliable scheduling and execution of health checks without requiring a persistent daemon.
+
+### Requirements for Automatic Health Checks
+
+- systemd must be available on the system
+- Container must not be running in rootless mode
+- Environment variable `DISABLE_HC_SYSTEMD` must not be set to "true"
```

Comment:
This should be a config property

https://github.com/containerd/nerdctl/blob/main/docs/config.md